### ISSUE TITLE
Redis lense doesn't support multiple IPs in bind command

### ISSUE THREAD

**Issue by yannh** [hercules-team/augeas#194]

**Comment by mfournier** [comment]
I have the following error when running the tests:
```
$ augparse -I lenses/ lenses/tests/test_redis.aug
Syntax error in lens definition
lenses/redis.aug:117.0-122.37:Failed to compile entry
lenses/redis.aug:117.12-121.15:exception: overlapping lenses in union.get
    Example matched by both: 'bind ::A
'
    First lens: lenses/redis.aug:117.12-120.18:
    Second lens: lenses/redis.aug:89.17-.101:
```

Tried figuring out the problem for a while, but I have to admit my augeas lens skills have faded away :-(

Hopefully someone on the team will be able to make sense of this error ?

**Comment by lutter** [comment]
The error message tries to say: there's an issue with the `entry` lens (defined on lines 117 column 0 through line 122 column 37). In particular, there's trouble with the piece from line 117 column 12 ("117.12") through line 121.15.

That expression does a lens union (`l1 | l2`) and those two lenses overlap (there is at least one string where both lenses would match) The first part of the problematic union is lines 117.12 through 120.18 (`standard_entry | save_entry | renamecmd_entry | slaveof_entry`), the second part is the `bind_entry` lens. When we see the string `bind ::A\n` we could use either of these two to process that string; Augeas does not allow that sort of ambiguity.

Looking a bit further, it seems that the culprit is `standard_entry`: the `entry_noempty` lens in there will match `bind ::A\n`. If I change `entry_noempty` to

    let entry_noempty = [ indent . key (k - "bind"). del_ws_spc
                         . Quote.do_quote_opt_nil (store v) . eol ]

all is well.

**Comment by opb1978** [comment]
just had this same problem. Would be great if this patch could merged into master!

**Comment by hedrickbt** [comment]
I have run into the same error on Ubuntu 18.04.2 LTS

line 69
bind 127.0.0.1 ::1


augtool> ls /augeas/files/etc/redis/redis.conf/error/
pos = 2755
line = 69
char = 14
lens/ = /usr/share/augeas/lenses/dist/redis.aug:120.10-.37:
message = Iterated lens matched less than it should

**Comment by raphink** [comment]
@hedrickbt 	this bug was fixed in #580 last year.
